src/main/java/uk/ac/ebi/intact/app/internal/utils/ModelUtils.java

```diff
@@ -179,11 +180,15 @@ public static void createUnknownColumnsFromIntactJSON(JsonNode elements, CyTable
                 String key = entry.getKey();
                 if (columnToType.containsKey(key)) return;
                 JsonNode value = entry.getValue();
-                if (value.isArray()) {
-                    columnToType.put(key, getJsonNodeValueClass(value.get(0)));
```

Comment:
`value.get(0)` was causing issues with empty arrays. I could had simply done:
```
if (!value.isNull()) {
  if (value.isArray()) {
    if (!value.isEmpty()) {
      columnToType.put(key, getJsonNodeValueClass(value.get(0)));
    }
  }
} 
```

But on the weird scenario where the value array is not empty but the first item is null (unlikely scenario), that could cause issues. So that's why I created `getArrayNodeValueClass`.